libs/feature/record/src/lib/data-downloads/data-downloads.component.ts

```diff
@@ -28,8 +28,8 @@ export class DataDownloadsComponent {
     switchMap((links) => {
       const wfsLinks = links.filter((link) => this.linkHelper.isWfsLink(link))
       const esriRestLinks = links
-        .filter(this.linkHelper.isEsriRestFeatureServer)
-        .flatMap(this.dataService.getDownloadLinksFromEsriRest)
+        .filter((link) => this.linkHelper.isEsriRestFeatureServer(link))
+        .flatMap((link) => this.dataService.getDownloadLinksFromEsriRest(link))
```

Comment:
Actually the real problem here is that this is not covered by the tests... do you think you can add a test for this? thanks :)

Reply:
I think we have some tests, but they return false positifs. I will try to improve this.